userhandles.go

```diff
@@ -74,21 +71,13 @@ func (cb *Corkboard) GetUser(w http.ResponseWriter, r *http.Request, ps httprout
 
 //RegisterUser is a HandlerFunc to deal with New User requests
 func (cb *Corkboard) RegisterUser() http.HandlerFunc {
```

Comment:
Do you need this? why not just pass in cb.CorkboardAuth.RegisterUser() to the router.